### ISSUE TITLE
Zecwallete Lite can't complete the SYNC

### ISSUE THREAD

**Issue by allexxey32** [adityapk00/zecwallet-lite#182]
Hello. I didn't use Zecwallete lite for about one month or so. Today I tried to enter the wallet and I realised that wallet can't complete the SYNC process. I have also tried to restore the wallet via seed phrase on my iphone - the sync process stoped on the mark of 80%, Now I see zero balance in the iphone app and "syncing process" which never ends on the dekstope app for windows. How to solve this problem?

**Comment by omar-nav** [comment]
Hi @allexxey32 I may have encountered the same issue. It's taking a lot longer than usual for the wallet to sync. It just took me almost ~15 minutes using cell phone hotspot. Make sure you have a reliable connection when syncing (It was taking way too long with my unreliable slow home WiFi. More than 30 minutes). Apparently, there's been a huge [increase in the average block size](https://blockchair.com/zcash/charts/average-block-size) 126x on the network over the past few days. The [limit for block size ](https://z.cash/support/faq/)is 2 MB

**Comment by adityapk00** [comment]
This is correct - There is an ongoing spam attack that is making blocks very big. Please connect to a reliable Wifi, and the sync will proceed.

**Comment by BostonZcash** [comment]
@allexxey32 Since you posted, Aditya's released some new versions which overcome various sync issues. Would recommend installing this release and giving it a new opportunity to sync: https://github.com/adityapk00/zecwallet-lite/releases/tag/v1.17.20
